pattern-library/sass/patterns/_forms.scss

```diff
@@ -260,12 +260,15 @@ $input-width: (
 
     // Input fields immediately followed by a button (ie, search boxes) should eliminate the spacing
     // between them and adopt matching styling/sizing to emphasize that they are connected.
-    &[type=search] + button {
+    + button {
         border-top-left-radius: 0;
         border-bottom-left-radius: 0;
         font-size: font-size(base);
         margin-left: (-1 * $baseline / 4);
```

Comment:
It's a [separate JIRA issue](https://openedx.atlassian.net/browse/FEDX-228) that this is know to be incompatible with RTL - if you wouldn't mind fixing it while you're in here (just needs an `@include margin-left`) we could fix both search box problems in the same release?